tools/topology/topology2/platform/intel/dmic-default.conf

```diff
@@ -36,4 +37,7 @@ Define {
 	DMIC_CORE_ID			0
 	PASSTHROUGH			"false"
 	DMIC_PIPELINE_PRIORITY		0
+	DMIC0_RATE                      48000
+	DMIC1_RATE                      16000
```

Comment:
DMIC0/1 rate are defined by spec or pipeline?

Reply:
I first thought that we would review DMIC with #8688 but maybe it's better that I close it and keep editing this.

So, these are the default values. We can specify other rates in build cmake if need. Does this look OK to you?